aggregator/src/compression/circuit.rs

```diff
@@ -174,6 +177,8 @@ impl CompressionCircuit {
         snark: Snark,
         has_accumulator: bool,
         rng: impl Rng + Send,
+        g2: &G2Affine,
```

Comment:
breaking changes in circuit API @silathdiir

`g2` and `s_g2` are from parameters